native/c/zam.c

```diff
@@ -190,7 +244,7 @@ int close_dcb(IHADCB *dcb)
   return rc;
 }
 
-int writeSync(IO_CTRL *ioc, char *buffer)
+int write_sync(IO_CTRL *ioc, char *buffer)
```

Comment:
lol, thanks for these...
Dumb question... should we enforce these going forward? 😋